app/controllers/chatfuel_controller.rb

```diff
@@ -1,13 +1,18 @@
 class ChatfuelController < ApplicationController
   def show
-    @topic = Topic.find_by(name: params[:topic])
-    if @topic
-      relevant_text_components = @report.active_chatbot_components.select {|c| c.topic_id == @topic.id}
-      @text_component = Text::Sorter.sort(relevant_text_components, {}).first
+    @report = Report.find_by_id(params[:report_id])
```

Comment:
this is done twice: https://github.com/roschaefer/story.board/blob/master/app/controllers/application_controller.rb#L16